docs/basics/use_nwbwidgets.ipynb

```diff
@@ -53,7 +53,7 @@
    "metadata": {},
```

Comment:
<p>make one paragraph, fix spelling typo 'specific'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I believe it already is one paragraph?